src/js/player.js

```diff
@@ -49,7 +49,12 @@ vjs.Player = vjs.Component.extend({
     this.language_ = options['language'] || vjs.options['language'];
 
     // Update Supported Languages
-    this.languages_ = options['languages'] || vjs.options['languages'];
+    var languages = vjs.obj.copy(options['languages'] || vjs.options['languages']);
+    vjs.options['languages'] = {};
```

Comment:
Are you taking the player level language definitions and copying them out to the `window.videojs` level here? I'm not following. Maybe a comment above would help.